docs/releasenotes/juju_2.9.x.md

```diff
@@ -7,7 +7,22 @@
 ```{note}
 Juju 2.9 series is LTS
 ```
-## 🔸 **Juju 2.9.52** - 07 July 2025
+### 🔸 **Juju 2.9.53**
+🗓️ 11 Dec 2025
```

Comment:
just wanted to confirm this, missed this earlier, this is 11 Dec because we are following UTC 0 timing right?